checkbox-provider-ce-oem/units/otg/jobs.pxu

```diff
@@ -1,134 +1,135 @@
-id: ce-oem-usb-otg/g_serial
-_summary: Check if USB OTG can work as serial.
+id: otg_ports
+plugin: resource
+_summary: Gather list of USB ports and UDC.
+_description:
+    A USB port and UDC mapping resource that relies on the user specifying in config varirable.
+    Usage of parameter: OTG={port1}:{node1} {port2}:{node2} ...
+    e.g. OTG=USB-C1:11200000 USB-Micro:112a1000
+estimated_duration: 1s
+environ: OTG
+flags: preserve-locale
+user: root
+command:
+    awk '{
+        split($0, record, " ")
+        for (i in record) {
+            split(record[i], data, ":")
+            cmd = "multiple-otg.sh -p " data[1] " -n " data[2]
+            system(cmd)
+        }
+    }' <<< "$OTG"
```

Comment:
I think it would be better if we move whole scripts into multiple-otg.sh, likes
multiple-otg.sh --resource "$OTG"

Reply:
Updated, thanks for the review.